package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "mongodb-realm-cli",
+  "version": "2.0.0-beta.0",
+  "description": "The MongoDB Realm Command Line Interface",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/10gen/realm-cli.git"
+  },
+  "license": "Apache-2.0",
+  "bin": {
+    "realm-cli": "./wrapper.js",
+    "transpiler": "./wrapper-transpiler.js"
+  },
+  "scripts": {
+    "install": "node install.js"
+  },
+  "keywords": [
+    "mongodb",
+    "realm"
+  ],
+  "author": "",
+  "dependencies": {
+    "request": "^2.88.0"
+  },
+  "devDependencies": {
+    "babel-eslint": "^8.2.2",
```

Comment:
⚠️  if we're not installing/building the transpiler anymore, are these dependencies needed?

Reply:
there are dependencies required for the install script, so we do need _some_ dependencies, but likely not all of these. probably worth going through and cleaning this up as a separate ticket.

Reply:
👍 ... filed https://jira.mongodb.org/browse/REALMC-8405